code/game/gamemodes/miniantags/abduction/machinery/abductor_camera.dm

```diff
@@ -89,7 +89,9 @@
 		return
 	var/obj/machinery/abductor/console/console = target
 
-	console.TeleporterRetrieve()
+	if(!console.TeleporterRetrieve())
+		to_chat(owner, "<span class='warning'>Error, unable to recall target. Please ensure they are not buckled, and that you have waited the required 10000 milliseconds!</span>")
+		playsound(owner, 'sound/machines/scanbuzz.ogg', 25, TRUE, SILENCED_SOUND_EXTRARANGE)
```

Comment:
I think this sound is a bit harsh, perhaps you could use terminal_error instead? 
I also think this 10000 millisecond stuff is a little bit too cute for it's own good, could you just use the remaining duration of the status effect?

Reply:
I would prefer a louder oh shit sound personally.